lib/watir.rb

```diff
@@ -14,58 +14,55 @@
 require 'watir/after_hooks'
 
 module Watir
+  @relaxed_locate = true
   @always_locate = true
 
+  attr_writer :relaxed_locate, :always_locate, :default_timeout, :prefer_css, :locator_namespace
+
   class << self
-    def always_locate?
-      @always_locate
+    #
+    # Whether or not Watir should wait for an element to be found or present
+    # before taking an action.
+    # Defaults to true.
+    #
+
+    def relaxed_locate?
+      @relaxed_locate
     end
 
     #
     # Whether or not Watir should cache element references or always re-locate an Element on use.
     # Defaults to true.
     #
 
-    def always_locate=(bool)
-      @always_locate = bool
-    end
-
-    def default_timeout
-      @default_timeout ||= 30
+    def always_locate?
+      @always_locate
     end
 
     #
     # Default wait time for wait methods.
     #
 
-    def default_timeout=(value)
-      @default_timeout = value
-    end
-
-    def prefer_css?
-      @prefer_css
+    def default_timeout
+      @default_timeout ||= 30
```

Comment:
Maybe we should differentiate the default timeout for implicit vs explicit waits. I am concerned that a default timeout of 30 seconds punish many users. 
* Consider a new user, who is learning Watir. They will likely get the element locators wrong on many occasions. However, each time it is wrong, they will see the browser sitting there for 30 seconds. This is a long enough time that the users will think that Watir has crashed or is slow.
* For those dealing with simpler pages (ie not JavaScript heavy pages), missing elements will greatly slow down test suites.

For implicit waits, I suggest a value closer to the 2 seconds, which is what Capybara uses.